doc/nrf/gs_ins_windows.rst

```diff
@@ -177,15 +179,24 @@ However, you might want to switch to a newer release or check out the latest sta
 To manage the ``nrf`` repository (the manifest repository), use Git.
 Checking out a branch or tag in the ``nrf`` repository gives you a different version of the manifest file.
 Running ``west update`` will then update the project repositories to the state specified in this manifest file.
+In order to obtain the latest changes, and regardless of the particular revision you want to check out, you will always need to :ref:`fetch the latest code <dm-wf-update-ncs>` from the GitHub `fw-nrfconnect-nrf`_ repository using ``git fetch``.
```

Comment:
This should be two lines up, because you always need to do it first.
"To make sure that you have the latest changes, run ``git fetch origin`` to :ref:...fetch the latest code... from the `fw-nrfconnect-nrf`_ repository.